apps/bot/src/app/commands/(module)/level/leaderboard.ts

```diff
@@ -83,7 +83,7 @@ export default new BotSubcommandBuilder()
 
       const leaderBoardCard = await generateLeaderboardCard({
         data: response,
-      }).then((res) => res.toAttachment())
+      }).toAttachment()
```

Comment:
you need to wrap the `generateLeaderboardCard` function call in brackets before you can run the `toAttachment` method